suite/tests/client-interface/drx-scattergather.dll.c

```diff
@@ -124,6 +125,31 @@ event_bb_analysis(void *drcontext, void *tag, instrlist_t *bb, bool for_trace,
     return DR_EMIT_DEFAULT;
 }
 
+static void
+search_for_next_gather_pc(void *drcontext, instr_t *start_instr, byte **gather_pc)
```

Comment:
Why not return the gather_pc?

Reply:
Done.

Reply:
Sorry, I forgot to add the commit to the last update. Not sure what happened. I will make this change along with the next PR, I need to add the scatter variants of the tests anyway, which will be the next patch.